src/modules/controlList.js

```diff
@@ -0,0 +1,98 @@
+const listContainer = document.querySelector('.list-container__items');
+const notification = document.querySelector('.notification');
+const roration = document.querySelector('.fa-arrows-rotate');
+const form = document.querySelector('.list-container__list-content');
+
+let tasks = [];
+
+const storage = localStorage.getItem('listItem');
+tasks = storage === null ? [] : JSON.parse(storage);
+
+const task = {
+  description: '',
+  completed: false,
+  index: 0,
+};
+
+// Loading...
+const spinner = () => {
+  roration.classList.add('spinner');
+  setTimeout(() => {
+    roration.classList.remove('spinner');
+  }, 2000);
+};
+
+const createElement = (element = '', className = [''], text = '', type = '', value = '', name = '') => {
+  const newElement = document.createElement(element);
+  newElement.classList.add(...className);
+  newElement.textContent = text;
+  newElement.type = type;
+  newElement.value = value;
+  newElement.name = name;
+  return newElement;
+};
+
+// Display tasks on eash list-item
+const displayTask = () => {
+  tasks.sort((a, b) => a.index - b.index);
+  tasks.forEach((e) => {
+    const i = createElement('i', ['icon', 'fa-solid', 'fa-ellipsis-vertical', 'bar']);
+    const input = createElement('input', ['checkbox'], '', 'checkbox');
+    const span = createElement('input', ['task-name'], '', '', e.description);
+    const li = createElement('li', ['list-container__items--item']);
+    li.append(input, span, i);
+    listContainer.appendChild(li);
+  });
+  notification.textContent = tasks.length;
+};
+
+displayTask();
+
+// Update Index after the list removal
+const updateIndex = () => {
+  for (let i = 0; i < tasks.length; i += 1) {
+    tasks[i].index = i;
```

Comment:
good job implementing the "for" loop :+1:  but it is a little bit old so you can use a "For each" loop

Reply:
It's really a good feedback, and I really appreciated, however, I just wanted to get the index number easily, that's why I used normal for loop.